python/feathub/processors/spark/spark_processor.py

```diff
@@ -35,6 +35,7 @@
     SparkProcessorConfig,
```

Comment:
Hi @Sxnan, could you please fix the todo in SparkProcessorConfig file and add a validator for `MASTER_CONFIG`? This TODO had been assigned to me according to offline discussions, but I think it is also proper for this PR to cover this TODO.